project/app/utils/complexity/squad_score.py

```diff
@@ -0,0 +1,100 @@
+import joblib
+import pandas as pd
+
+
+def metrics(transcription: str):
```

Comment:
Can you call this method from squad_score and change the parameter so that it's `squad_score(document: str)` so that its a plug n play solution for the endpoints.

Reply:
Completed in recent commit.